types/prismjs/index.d.ts

```diff
@@ -142,10 +142,14 @@ export interface LanguageDefinition {
     rest?: Token[];
 }
 
-export interface Languages {
+export type Languages
```

Comment:
Why not write this as `export interface Languages extends LanguageMap, LanguageProtocol {}`? I know it's stylistic here, but `interface`s tend to be more efficient internally.